### PR TITLE
add danish README

### PR DESCRIPTION
**Pull request by salkin-mada**

### CODE REVIEW

File: README.dk.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+## Udgivelses Signaturer
+
+Udgivelser er signeret med [B22DA89A](http://pgp.mit.edu/pks/lookup?op=vindex&search=0x52CB6609B22DA89A)
+og publiseret på [GitHub](https://github.com/swaywm/sway/releases).
+
+## Installation
+
+### Fra Pakker
+
+Sway er tilgængelig i mange distributioner. Prøv at installere pakken "svaje" til
```
Comment (ivesen) [inline README.dk.md]:
```suggestion
Sway er tilgængelig i mange distributioner. Prøv at installere pakken "sway" til
```
I assume you didn't intend to translate sway :)

File: README.dk.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
+# Sway
+
+Sway er en [i3](https://i3wm.org/)-kompatibel [Wayland](http://wayland.freedesktop.org/) typograf.
+Læs [Ofte stillede spørgsmål](https://github.com/swaywm/sway/wiki).
+Deltag på [IRCkanalen](http://webchat.freenode.net/?channels=sway&uio=d4) (#sway på irc.freenode.net).
+
+Hvis du har lyst til at støtte sway udviklingen, venlist send bidrag på [SirCmpwn'sPatreon side](https://patreon.com/sircmpwn).
```
Comment (Sviundt) [inline README.dk.md]: Is it intentional to not use spaces in the [SirCmpwn'sPatreon side] link text?

**Comment by Pectojin** [comment]
The "til" at the end of 'Prøv at installere pakken "sway" til' makes it look like the sentence was cut off. If it's the end of the sentence "til" should just be omitted.

Also "typografi" is a bad translation of "compositor". I had no idea what it meant because it translates back to "typography". I would probably opt for treating "wayland compositor" as imported words and leaving them in the English. It's a fairly common practice in contemporary Danish, especially in tech.

File: README.dk.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+På systemer uden 'logind', behøver du at sætte ejerens bruger-id for Sways eksekverbare filer - såkaldt SUID (Set owner User ID):
+
+    sudo chmod a+s /usr/local/bin/sway
+
+Sway vil frasige sig 'root' tilladelser kort efter opstart
+
+## Konfiguration
+
+Hvis du allerede bruger i3, bør du kopiere din i3-konfiguration til `~/.config/sway/config` og
+det vil fungerer ud af kassen. Ellers skal du kopiere eksempel konfigurations filen til
```
Comment (Pectojin) [inline README.dk.md]: "fungerer ud af kassen" doesn't really translate. We do use "out of the box" as an expression in Danish, but you could also opt for "og det vil bare fungerer"

File: README.dk.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+`~/.config/sway/config`. Den er normalt placeret i `/etc/sway/config`.
+Kør `man 5 sway` for at få oplysninger om konfigurationen.
+
+## Kører
+
+Kør `sway` fra en TTY. Nogle skærm ledere (display managers) fungerer muligvis, men understøttes ikke af
```
Comment (Pectojin) [inline README.dk.md]: I wouldn't bother translating "display managers" here. I've never ever heard a Danish person not call it a display manager, so I doubt anyone would understand the Danish translation.

File: README.dk.md (hunk 1 above)
Comment (salkin-mada) [reply README.dk.md]: indeed i did not :)

File: README.dk.md (hunk 2 above)
Comment (salkin-mada) [reply README.dk.md]: non intentional, thanks for noticing.
will --amend with a g in venligst.

File: README.dk.md (hunk 3 above)
Comment (salkin-mada) [reply README.dk.md]: agree. will --amend

File: README.dk.md (hunk 4 above)
Comment (salkin-mada) [reply README.dk.md]: true that, --amend

**Comment by ddevault** [comment]
Good to go?

**Comment by salkin-mada** [comment]
Yepp

**Comment by ddevault** [comment]
Thanks!

**Pull request merged**